libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -155,9 +155,19 @@ class AP_BoardConfig {
         UNLOCK_FLASH = (1<<4),
         WRITE_PROTECT_FLASH = (1<<5),
         WRITE_PROTECT_BOOTLOADER = (1<<6),
-        SKIP_BOARD_VALIDATION = (1<<7)
+        SKIP_BOARD_VALIDATION = (1<<7),
+        DISABLE_ARMING_GPIO = (1<<8)
```

Comment:
I think this should be enable rather than disable, otherwise people's configs are going to break when they upgrade. For instance I wouldn't want this set to the VTX GPIO pin because many people solve this via SmartAudio which already has support for coming out of pitmode on arming

Reply:
this cant break existing setups unless someone intentionally changes a hwdef to do so by adding the feature to something they should not...this is primarily intended to provide new functionality for pins internal to the board for telem module rf control....and requiring it to need a param change for intended automatic functionality, instead of an option to override in special cases, is undesirable..
this is for the ATOMRC and SpeedyBee BT modules whic currently dont have this desired automatic power control..

Reply:
@andyp1per I agree with Henry, this can't impact existing boards

Reply:
It can't - I agree, but what I don't want is for it to be used to control VTX power by default on new boards - we already have a function for that and this will conflict with that. Is it going to be used for VTX power?

Reply:
If it *is* going to be used for VTX power then we should integrate with the existing option rather than creating a new on-by-default alternative.

Reply:
it isnt

Reply:
ok